### PR TITLE
root withdraw

### PR DESCRIPTION
**Pull request by kbrav**

### CODE REVIEW

File: core/root.sol [hunk 1]
```diff
@@ -30,14 +33,20 @@ contract RootZone {
...
-        user.call{gas: 2300, value: pile}("");
+        back[user] += pile;
         pile = msg.value;
         user = msg.sender;
         dark = hash;
         term = block.timestamp + FREQ;
         emit Ante(pile);
     }
 
+    function withdraw() external {
```
Comment (stobiewan) [inline core/root.sol:44]: We should add a reentrancy test for withdraw() and probably set back[user] to zero before the refund
